hw/ip/edn/data/edn.hjson

```diff
@@ -603,9 +606,7 @@
         { bits: "30",
           name: "FIFO_STATE_ERR",
           desc: '''
-                This bit will be set to one when any of the source bits (bits 0 through 1 of this
-                this register) are asserted as a result of an error pulse generated from
-                any FIFO where both the empty and full status bits are set.
+                This bit will be set to one when any of the source bits (bits 0 through 1 of this register) are asserted as a result of an error pulse generated from any FIFO where both the empty and full status bits are set or in case of error conditions inside the hardened counters.
```

Comment:
I don't believe we've got a hard "one sentence to a line" rule for hjson files, so I'm not sure this is necessary. No problem, but it might be more comfortable to edit without the unwrapping step.

Reply:
You're right, we don't have a hard rule here (wasn't aware we have a separate style guide for hjson, see https://opentitan.org/book/doc/contributing/style_guides/hjson_usage_style.html) but I always tell people to do 1 sentence per line to ease future diffs.